tools/ipython/ipython.py

```diff
@@ -0,0 +1,4 @@
+import ipdb
+
+if __name__ == "__main__":
+    ipdb.set_trace()
```

Comment:
Could instead use `from IPYthon.terminal import interactiveshell,prompts` and `TerminalInteractiveShell().run_cell()`